WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -142,4 +160,22 @@ public class AppSettingsViewController: UITableViewController {
             self.navigationController?.pushViewController(controller, animated: true)
         }
     }
+
+    func openApplicationSettings() -> ImmuTableAction {
+        return { row in
```

Comment:
If the notification observers were added just to deselect the tableview row when resuming the app, maybe instead just deselect the row before returning and remove the code related to the notification observers. 

Bonus, it will mean the cell deselects even if the `guard` fails (tho I can't think of why that should ever happen).